services/skus/controllers.go

```diff
@@ -600,6 +601,42 @@ func CreateOrderCreds(svc *Service) handlers.AppHandler {
 	}
 }
 
+// deleteItemCreds handles requests for deleting credentials for an item.
+func deleteItemCreds(svc *Service) handlers.AppHandler {
+	return func(w http.ResponseWriter, r *http.Request) *handlers.AppError {
+		ctx := r.Context()
+		lg := logging.Logger(ctx, "skus").With().Str("func", "deleteItemCreds").Logger()
+
+		orderID, err := uuid.FromString(chi.URLParamFromCtx(ctx, "orderID"))
+		if err != nil {
+			lg.Error().Err(err).Msg("failed to validate order id")
+			return handlers.ValidationError("request url parameter", map[string]interface{}{
+				"orderID": err.Error(),
+			})
+		}
+
+		reqID, err := uuid.FromString(chi.URLParamFromCtx(ctx, "reqID"))
+		if err != nil {
+			lg.Error().Err(err).Msg("failed to validate request id")
+			return handlers.ValidationError("request url parameter", map[string]interface{}{
+				"itemID": err.Error(),
+			})
+		}
+
+		isSigned := r.URL.Query().Get("isSigned") == "true"
+
+		if err := svc.DeleteOrderCreds(ctx, orderID, reqID, isSigned); err != nil {
+			if errors.Is(err, model.ErrOrderNotFound) || errors.Is(err, ErrOrderHasNoItems) {
+				return handlers.WrapError(err, "order or item not found", http.StatusNotFound)
+			}
+			lg.Error().Err(err).Msg("failed to delete the order credentials")
+			return handlers.WrapError(err, "error deleting credentials", http.StatusInternalServerError)
+		}
```

Comment:
question: should the `errMaxActiveOrderCredsExceeded` error be handled somehow? Does need a non-500 response? Will the client have to be able to understand this case?

Reply:
Addressed in https://github.com/brave-intl/bat-go/pull/2325.